### DISCUSSION TITLE
Changing amount of loaded search results

### DISCUSSION THREAD

**Discussion by wofsauge** [squidfunk/mkdocs-material#2317] [Q&A] [answered]
Hi,

On my documentation page, i changed the layout of the search results, in order to make them more compact and list more informations. Here is an example of how it looks: 
![image](https://user-images.githubusercontent.com/245725/108631335-c3c83c80-7469-11eb-826c-a686b046848f.png)

**My Problem**
My problem right now is, that it only displays the first 10 results and only loads the next one by one while scrolling down. This behavior combined with the compact layout causes the results to load really unsmoothly while scrolling (Jumping, Jittering, etc.).

**Is there a way to change the amount of search results initially loaded, or the amount of entries loaded while scrolling down?**

Or is there maybe a way to force it to load more entries?

Thanks for the help already :)

**Comment by squidfunk** [comment, marked as answer]
Could you please try the latest beta #2290? Search result loading has been improved as part of this release.

**Reply by wofsauge** [reply]
That build fixes the jittering with my layout and it seems to load 14 instead of 10 entries :)

Its definetly much better than before. I guess i will use that build for now ;)

Thank you very much for your reply
